### DISCUSSION TITLE
Attiny85 sketch not compile in IDE 2.3.2 but OK with IDE 1.8.19

### DISCUSSION THREAD

**Discussion by pierrotm777** [SpenceKonde/ATTinyCore#873] [Support / Q & A]
Hello,

I try to compile a sketch for an Attiny85 with bootloader.
My error is:
`
FQBN: ATTinyCore:avr:attinyx5:clock=737external
Using board 'attinyx5' from platform in folder: C:\Users\pierrot\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2
Using core 'tiny' from platform in folder: C:\Users\pierrot\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2

Detecting libraries used...
C:\Users\pierrot\AppData\Local\Arduino15\packages\DxCore\tools\avr-gcc\7.3.0-atmel3.6.1-azduino4/bin/avr-g++ -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -flto -w -x c++ -E -CC -mmcu=attiny85 -DF_CPU=7372800L -DCLOCK_SOURCE=1 -DARDUINO=10607 -DARDUINO_AVR_ATTINYX5 -DARDUINO_ARCH_AVR -DNEOPIXELPORT=PORTB -IC:\Users\pierrot\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny -IC:\Users\pierrot\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\variants\tinyX5 C:\Users\pierrot\AppData\Local\Temp\arduino\sketches\56369B200F4ABC19401902040B023632\sketch\FrSkyMagnetometer.ino.cpp -o nul
exec: "C:\\Users\\pierrot\\AppData\\Local\\Arduino15\\packages\\DxCore\\tools\\avr-gcc\\7.3.0-atmel3.6.1-azduino4/bin/avr-g++": file does not exist

Compilation error: exec: "C:\\Users\\pierrot\\AppData\\Local\\Arduino15\\packages\\DxCore\\tools\\avr-gcc\\7.3.0-atmel3.6.1-azduino4/bin/avr-g++": file does not exist`

I send you my code.

Thanks for your help :-)

[FrSkyMagnetometer.zip](https://github.com/user-attachments/files/16626351/FrSkyMagnetometer.zip)

**Comment by SpenceKonde** [comment]
Core not installed properly or compiler toolchain included has been
mistaken for a virus by anti-virus and the compiler was dumped into
quarantine, as shown by the fact that the compiler is file not found.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Comment by pierrotm777** [comment]
I use the json file for include the attiny core.
The error is for Dxcore, does exist a json file for this Dxcore?

**Comment by pierrotm777** [comment]
Hello,

I have solved my issue by installing the DxCore with the json link http://drazzy.com/package_drazzy.com_index.json .
